alf/networks/actor_networks.py

```diff
@@ -134,6 +134,8 @@ def forward(self, observation, state=()):
 
         observation, state = super().forward(observation, state)
         encoded_obs, _ = self._encoding_net(observation)
+        if isinstance(info, dict):
+            info["pre_activation"] = encoded_obs
```

Comment:
In principle shouldn't change the code of actor_networks.py. Should only modify ddpg's code.